tests/components/caldav/test_calendar.py

```diff
@@ -774,3 +774,32 @@ async def test_event_rrule_hourly_ended(mock_now, hass, calendar):
     state = hass.states.get("calendar.private")
     assert state.name == calendar.name
     assert state.state == STATE_OFF
+
+
+async def test_get_events(hass, calendar):
+    """Test that all events are returned on API."""
+    assert await async_setup_component(hass, "calendar", {"calendar": CALDAV_CONFIG})
+    await hass.async_block_till_done()
+    entity = hass.data["calendar"].get_entity("calendar.private")
```

Comment:
Please test the `async_get_events` method via the http api. See for an example how to test a http view:

https://github.com/home-assistant/core/blob/14c205384171dee59c1a908f8449f9864778b2dc/tests/components/zwave_js/test_api.py#L158-L167

Reply:
:+1: I'll try and find time this weekend to update tests.